src/balanceTransfer.ts

```diff
@@ -1,81 +1,31 @@
 import Metadata from '@polkadot/metadata';
 import { createType, TypeRegistry } from '@polkadot/types';
-import { SignerPayloadJSON } from '@polkadot/types/types';
 
 import { EXTRINSIC_VERSION, ONE_SECOND } from './util/constants';
+import { BaseTxInfo, UnsignedTransaction } from './util/interfaces';
```

Comment:
We often do that too in PolkadotJS projects: put all interfaces in a separate file. Could you rename the file to `types.ts` to be consistent with PolkadotJS?